memgpt/schemas/job.py

```diff
@@ -0,0 +1,38 @@
+import uuid
+from datetime import datetime
+from enum import Enum
+from typing import Optional
+
+from pydantic import BaseModel, Field
+
+from memgpt.utils import get_utc_time
+
+
+class JobStatus(str, Enum):
```

Comment:
🪰 **nitt** - I like to move all my enums into a single `enums` module for organization, but some people treat them like a primitive and just have them wherever. Your call